.ci/mount-volume/simple-pod.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# First parameter either "Start" or "Stop"
+function podStarted {
+  POD_NAME=$1
+  SIMPLE_POD_JSON=$(oc get pod "$POD_NAME" -o json)
+  RETURN_CODE=$?
+
+  if [[ $RETURN_CODE -ne 0 ]]; then
+    echo "Could not obtain information about pod. Trying again."
+    return 1
+  fi
+
+  POD_STATUS=$(echo "$SIMPLE_POD_JSON" | jq --raw-output '.status.phase')
+
+  echo "Wanted: Running     Actual: ${POD_STATUS}"
+  if [[ $POD_STATUS == "Running" ]]; then
+    echo "Pod is running."
+    return 0
+  else
+    return 1
+  fi
+}
+
+function podStopped {
+  POD_NAME=$1
+  if oc get pod "$POD_NAME" -o json > /dev/null 2>&1; then
+    # pod found, not deleted yet.
+    return 1
+  else
+    return 0
+  fi
+}
+
+function waitForPodToBeRunning {
+  echo "Waiting for pod to start"
+  start=$(($(date +%s%N)/1000000))
+  TIMEOUT_IN_MILLISEC=$(( $ATTEMPT_TIMEOUT * 1000))
+  while [[ ${CURRENT_TRY_TIME} -le ${TIMEOUT_IN_MILLISEC} ]]; do
+    end=$(($(date +%s%N)/1000000))
+    CURRENT_TRY_TIME=$(expr $end - $start)
+    if podStarted "$POD_NAME"; then
+      echo $CURRENT_TRY_TIME >> Start.csv
+      return
+    else
+      sleep 1
+    fi
+  done
+  echo "Waiting for pod to change its state timed out. Exiting."
+  exit 1
+}
+
+function waitForPodToStop {
+  echo "Waiting for pod to be removed"
+  start=$(($(date +%s%N)/1000000))
+  TIMEOUT_IN_MILLISEC=$(( $ATTEMPT_TIMEOUT * 1000))
+  while [[ ${CURRENT_TRY_TIME} -le ${TIMEOUT_IN_MILLISEC} ]]; do
+    end=$(($(date +%s%N)/1000000))
+    CURRENT_TRY_TIME=$(expr $end - $start)
+    if podStopped "$POD_NAME"; then
+      echo $CURRENT_TRY_TIME >> Stop.csv
+      return
+    else
+      sleep 1
+    fi
+  done
+  echo "Waiting for pod to change its state timed out. Exiting."
+  exit 1
+}
+
+function simplePodRunTest {
+  COUNTER=1
+  echo "Number of iterations: ${ITERATIONS}"
+
+  SIMPLE_POD_CONFIGURATION_JSON=$(jq ".spec.volumes[].persistentVolumeClaim.claimName |= \"$VOLUME_NAME\"" simple-pod.json)
+
+  while [[ ${COUNTER} -le ${ITERATIONS} ]]; do
+    echo "ITERATION #${COUNTER}"
+    echo "$SIMPLE_POD_CONFIGURATION_JSON" | oc apply -f -
```

Comment:
You've removed the second call. Have you tried that it works now?

Reply:
I thought were were still discussing the necessity of the call. I can put it back for now so that we don't delay this PR anymore and then I can fix it as part of my refactoring task

Reply:
added the line back

Reply:
Does it mean that you didn't try that or that it didn't work?

Reply:
I didn't try it.

Reply:
I will fix it as a part of the refactoring task